README.md

```diff
@@ -1 +1,59 @@
-# Hello-World
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello World
+
+> A basic Hello Microverse Project with HTML and CSS. It has Linters Implemented and ignored redundant files using .gitignore
+
+![screenshot](./app_screenshot.png)
+
```

Comment:
- [ ] The screenshot link is broken Kindly fix it😉

Reply:
It has been fixed. Thank you.